packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js

```diff
@@ -168,6 +178,27 @@ export default class QuanticSearchInterface extends LightningElement {
     );
   }
 
+  bindAriaLiveEvents() {
```

Comment:
It seems to me like we'll want this capability on other interface types such as Insight and Case Assist and these handlers seem generic enough so perhaps they could be in some sort of Util to be easily reused?

Reply:
You mean create like a generic utils method to handle these two events?
Something like this?

`Utils.dispatchMessageHandler(element, event);`

You need to have a reference to the current element since you need to be able to find the aria-live component inside your own DOM

Reply:
Since I need the element itself I think I would rather keep it directly here? And add new ones for Case Assist and Insight? what do you think?

Reply:
When we would add new ones it would be a straight up copy paste though no? 🐟

Reply:
Not the end of the world I guess just think it would make future development less error-prone.

Reply:
Not sure I follow can you flesh out the suggestion?